sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v3/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Ok I think this just got mixed up in the naming of things. This friendly name and description here should go in the metadata for the other file and probably v.v..

Reply:
Indeed 😳